web/templatetags/web_extras.py

```diff
@@ -114,9 +117,14 @@ def nav_link(request, url_name, text, html_classes=None, queryString=None):
     if queryString:
         url = url + queryString
 
-    return mark_safe(
-        f'<a class="{" ".join(html_classes)}"{aria_current} href="{url}">{_(text)}</a>'
-    )
+    if list:
+        return mark_safe(
+            f'<li class="nav-item"><a class="{" ".join(html_classes)}"{aria_current} href="{url}">{_(text)}</a></li>'
+        )
+    else:
+        return mark_safe(
+            f'<a class="{" ".join(html_classes)}"{aria_current} href="{url}">{_(text)}</a>'
+        )
```

Comment:
The a-tag element is the same in both cases.  This isn't critical, but it might be better to save the a-tag to a variable and wrap it in a li-tag if list is True.

Reply:
Yep that is a better way. Thanks!